sdk/core/azure-core/src/main/java/module-info.java

```diff
@@ -34,6 +34,7 @@
     opens com.azure.core.util to com.fasterxml.jackson.databind;
     opens com.azure.core.util.logging to com.fasterxml.jackson.databind;
     opens com.azure.core.util.serializer to com.fasterxml.jackson.databind;
+    opens com.azure.core.models.spatial to com.fasterxml.jackson.databind;
```

Comment:
I don't believe this is needed, opening to databind is only needed when using reflective code to serialize and deserialize objects. The spatial models use custom serialization and deserialization which doesn't perform any reflect accesses.

Reply:
I am having for tests. Forgot to remove. Thanks for catching this!